index.html

```diff
@@ -2309,6 +2309,112 @@ <h2 id="AudioDestinationNode">
               not be changed.
             </p>
           </dd>
+          <dt>
+            readonly attribute double latency
```

Comment:
Since we also now have baseLatency (for the playback category stuff), latency is, perhaps, too generic.  Maybe outputLatency, or deviceOutputLatency, or something else?